cobbler/settings.py

```diff
@@ -100,7 +100,7 @@
     "mgmt_parameters": [{}, "dict"],
     "next_server": ["127.0.0.1", "str"],
     "nsupdate_enabled": [0, "bool"],
-    "power_management_default_type": ["ipmitool", "str"],
+    "power_management_default_type": ["ipmilan", "str"],
```

Comment:
what is `ipmilan`? This seems like an unrelated change...

Reply:
It is indeed unrelated to this PR, but it keeps on popping up while testing.
See https://gitter.im/cobbler/community?at=5e32e80715941335583ee67a

Reply:
This is correct. This change is lgtm.

Reply:
Reverted because of #2267